### PR TITLE
Feature/create pet page sa

### PR DESCRIPTION
**Pull request by StevieAmb**
### 1. What changed?
The site page for the single pet detail was created, and hard-coded with dummy data so that styling could be applied to different parts of the card.

### 2. Why is this change necessary?
This change was necessary because after the user chooses what pet they want to adopt from the home page, and click on an individual pet, they need to have a page that gives them more information about that individual. So, we need to create a page that has the additional details about the pet.

### 3. What changed technically that may impact others?
The additional page will need to be added to the routes via the click of the image on the home page, and so we need to add the page to routes.

### 4. How do we test it?
We can test it by running the cypress test, and then click the spec that is called `singlePetPage_spec.js` and see that all of the tests for what is being displayed on the page pass.

### CODE REVIEW

File: cypress/integration/singlePetPage_spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+describe('Pet Applications Component', () => {
```
Comment (anewb87) [inline cypress/integration/singlePetPage_spec.js:1]: Nice testing. Checking to make sure everything's displayed on the page.

File: src/Components/Pet.js [hunk 2]
```diff
@@ -1,9 +1,66 @@
 import React from 'react';
 import '../Styles/Pet.scss';
+import Header from './Header';
+
+//Posting happens before querying
```
Comment (Fordo29) [inline src/Components/Pet.js:5]: Probably want to remove the pseudocode.

File: cypress/integration/singlePetPage_spec.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      .get('.single-pet-type')
+      .contains('French Bull Dog')
+      .get('.single-pet-description')
+      .contains("Hi, I'm Stewy!")
+      .get('img')
+      .should('exist')
+      .get('.view-app-button')
+      .contains('View Applications')
+      .get('.submit-app-button') 
+      .contains('Submit Application')
```
Comment (Fordo29) [inline cypress/integration/singlePetPage_spec.js:25]: Nice testing for the entire page display.

File: src/Components/Pet.js [hunk 4]
```diff
@@ -1,9 +1,66 @@
...
+          <div className='middle-o-card-container'>
+            <div className='pet-details'>
+              <div className='light-pet-details'>
+                <h2 className='single-pet-name'>Stewart</h2>
+                <p className='single-pet-age'>9 years old</p>
+                <p className='single-pet-gender'>Female</p>
+                <p className='single-pet-type'>French Bull Dog</p>
+              </div>
+              <div className='single-pet-description'>
+                <p>Hi, I'm Stewy! I'm so happy to see you! I really like warm sweaters and belly rubs!
```
Comment (anewb87) [inline src/Components/Pet.js:33]: All of this repetitive text is to assist with styling the page. We have to assume that there will be longer descriptions input by user's so we're accounting for that now with this hard-coded information.

File: src/Styles/Pet.scss [hunk 5]
```diff
@@ -0,0 +1,93 @@
+@import '_variables.scss';
```
Comment (anewb87) [inline src/Styles/Pet.scss:1]: This styling looks really good. Inline with  the rest of the app.

**Review by anewb87 — APPROVED** [review]
Awesome. Moving along. This was, I believe, the last component that really need/could've been built out. We are ready to get some data and start making queries.

**Review by ectweitmann — APPROVED** [review]
This looks good. It'll be nice if we have the time to go back and standardize where/when we use single/double quotes. But that is a low priority.

File: src/Components/Pet.js [hunk 6]
```diff
@@ -2,15 +2,6 @@ import React from 'react';
 import '../Styles/Pet.scss';
 import Header from './Header';
 
-//Posting happens before querying
```
Comment (anewb87) [inline src/Components/Pet.js]: Good call on deleting the commented out code and pushing before the merge.

**Review by anewb87 — APPROVED** [review]
Cleaning up commented out code before merge.

File: src/Components/Pet.js (hunk 2 above)
Comment (anewb87) [reply src/Components/Pet.js]: And done. Stevie cleaned it up and pushed before the merge

**Review by Fordo29 — APPROVED** [review]

**Pull request merged**